shared/chat/inbox/container/remote.js

```diff
@@ -22,8 +22,31 @@ const valuesCached = memoize((...vals) => vals.map(v => v))
 
 const metaMapToFirstValues = memoize(metaMap =>
   metaMap
+    .filter((meta, id) => {
+      if (Constants.isValidConversationIDKey(id)) {
+        if (meta.teamType === 'adhoc') {
+          // DM's
+          return !meta.isMuted ? true : false
+        } else if (meta.teamType === 'big') {
+          // channels
+          if (!meta.notificationsGlobalIgnoreMentions) {
+            // @here and @mention not ignored
+            if (meta.notificationsDesktop === 'onAnyActivity') {
+              return true
+            } else if (meta.notificationsDesktop === 'onWhenAtMentioned') {
+              // check if user has been mentioned in the message
+              let snippet = meta.snippet.toLowerCase()
+              return snippet.indexOf(`@${_username}`) === 0 || snippet.indexOf('you') === 0
```

Comment:
since `_username` is gone now, this needs to be
```javascript
return (!!_laststate.config.username && snippet.indexOf(`@${_laststate.config.username}`) === 0) || snippet.indexOf('you') === 0
```